server/plugin/api.go

```diff
@@ -1094,8 +1175,16 @@ func (p *Plugin) createIssue(w http.ResponseWriter, r *http.Request, userID stri
 	}
 
 	ghIssue := &github.IssueRequest{
-		Title: &issue.Title,
-		Body:  &issue.Body,
+		Title:     &issue.Title,
+		Body:      &issue.Body,
+		Labels:    &issue.Labels,
+		Assignees: &issue.Assignees,
+	}
+
+	// submitting the request with an invalid milestone ID results in a 422 error
+	// we make sure it's not zero here, because the MM client might have left this field empty
```

Comment:
Nit pick: What is "the MM client"? Can the documentation be more specific.

Reply:
@hanzei I was referring to the front-end HTTP client in this plugin: https://github.com/mattermost/mattermost-plugin-github/blob/master/webapp/src/client/client.js

I could reword "MM client" as "front-end client" unless you have another suggestion?

Reply:
`webapp client` or just `webapp` is a term often used in MM. But `front-end client` is also fine.